README.md

```diff
@@ -209,6 +209,24 @@ select * from {{ ref('users') }}
 {% endif %}
 ```
 
+##### `merge`
+
+Through the `merge` incremental strategy, dbt-trino constructs `MERGE` statement which `INSERT` new records and `UPDATE` existing ones based on unique key specified by `unique_key`. 
```

Comment:
Also add a note that they can use the `delete+insert` materialization if their `unique_key` is not unique